### PR TITLE
Backport translator comment PR to 3.2

### PR DESCRIPTION
**Pull request by SkyLucilfer**
Backport PR #41510 to 3.2 branch.

### CODE REVIEW

File: editor/translations/extract.py [hunk 1]
```diff
@@ -100,7 +220,7 @@ def process_file(f, fname):
 print("Updating the editor.pot template...")
 
 for fname in matches:
-    with open(fname, "r") as f:
+    with open(fname, "r", encoding="utf8") as f:
```
Comment (akien-mga) [inline editor/translations/extract.py:223]: This will break Python 2 compatibility, you should use `open_utf8` from our `compat.py` (`import compat`).

Comment (akien-mga) [reply editor/translations/extract.py:223]: Ah nevermind, this is not part of the buildsystem so it doesn't matter.

But might be worth bumping the shebang to `python3` then.

**Review by akien-mga — APPROVED** [review]

File: editor/translations/extract.py (hunk 1 above)
Comment (SkyLucilfer) [reply editor/translations/extract.py:223]: Ahh ok. Didn't see your msg. I will revert the changes.

Comment (SkyLucilfer) [reply editor/translations/extract.py:223]: @akien-mga You want me to change to `#!/usr/bin/env python3`? I didn't understand the shebang part until now. 
The `/usr/bin/env` part I refer from this [discussion](https://stackoverflow.com/questions/22222473/shebang-doesnt-work-with-python3).

Comment (akien-mga) [reply editor/translations/extract.py:223]: Yeah that works fine.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**